pyproject.toml

```
@@ -34,6 +34,7 @@ urls.changelog = "https://pypa-build.readthedocs.io/en/stable/changelog.html"
 dependencies = [
   "packaging >= 19.0",
   "pyproject_hooks",
+  "pip>=22.3",
```

```suggestion
  "pip>=22.3",
  "pyproject_hooks",
```

Just keeping things roughtly alphabetically sorted